pkg/build/golang/deps.go

```diff
@@ -0,0 +1,83 @@
+package golang
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/archive"
+)
+
+func parseDependencies(raw string) map[string]string {
```

Comment:
This is a good candidate for a unit test with a few test cases. For example I see we have a version where version stays empty, but I am not sure this can be the case with `go mod`. I believe you always get a version, even if it is pseudo-version.